From 3411ce7970f01f036a66f39e7c72f6e44672a455 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 21 Jan 2021 14:22:42 +0200
Subject: [PATCH] [SW-30548] i2c: change init/disable prints to dbg

uart key press handling changed to slow_return, so on every key press
i2c module disable/init messages are emitted to console.
changing those prints to debug prints in order to reduce console
clutter.

Change-Id: I628507107739b510a1b9c1a5b3eb2fb7d7dca65a
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/common/module_i2c_slave.c b/common/module_i2c_slave.c
index 7e4715c..feebccb 100644
--- a/common/module_i2c_slave.c
+++ b/common/module_i2c_slave.c
@@ -149,7 +149,7 @@
 		i2cslave_flush();
 	}
 
-	preboot_puts("I2C SLAVE initialized\n");
+	preboot_puts_dbg("I2C SLAVE initialized\n");
 }
 
 enum cmd_check_res module_i2c_s_check(void)
@@ -186,7 +186,7 @@
 	regmap_writel(0x0, mmPSOC_I2C_S_IC_ENABLE);
 	i2cslave_flush();
 
-	preboot_puts("I2C SLAVE: disabled\n");
+	preboot_puts_dbg("I2C SLAVE: disabled\n");
 }
 
 u8 i2cslave_pcie_vendor_id_cmd(u8 cmd)
